clients/spark/core/src/main/scala/io/treeverse/clients/LakeFSInputFormat.scala

```diff
@@ -180,8 +180,11 @@ class LakeFSCommitInputFormat extends LakeFSBaseInputFormat {
     val ranges = commitIDs
       .flatMap(commitID => {
         val metaRangeURL = apiClient.getMetaRangeURL(repoName, commitID)
-        val rangesReader = metarangeReaderGetter(job.getConfiguration, metaRangeURL, true)
-        read(rangesReader).map(rd => new Range(new String(rd.id), rd.message.estimatedSize))
+        if (StringUtils.isBlank(metaRangeURL)) None
```

Comment:
Unless metaRangeURL can be null (AFAIK it never is, and I generally dislike conflating null and "" as the same!), I think we can just say:

```suggestion
        if (metaRangeURL == "") None
```